tools/ci_build/github/linux/docker/Dockerfile.ubuntu_openvino

```diff
@@ -1,7 +1,7 @@
 ARG UBUNTU_VERSION=20.04
 FROM ubuntu:${UBUNTU_VERSION}
 
-ARG OPENVINO_VERSION=2022.1.0
+ARG OPENVINO_VERSION=2022.2.0
 ARG PYTHON_VERSION=3.8
```

Comment:
Though this file supports customizing python versions, but do you need to use different python versions in olive tests? If not, please try to use the default one provided by Ubuntu. For example, the default one for Ubuntu 20.04 is python 3.8.

Reply:
change default python version to 3.8